### PR TITLE
Create PT-BT version of platform-as-a-service.md

### LINKED ISSUE #1216: [Portuguese] Localize 'Platform as a Service (PaaS)'
Localize [https://glossary.cncf.io/platform-as-a-service/](https://glossary.cncf.io/platform-as-a-service/)

### PR DESCRIPTION
**Pull request by juliafmorgado**
Signed-off-by: Julia Furst Morgado <52685951+juliafmorgado@users.noreply.github.com>

### Describe your changes
Create PT-BT version of platform-as-a-service.md

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1216

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by juliafmorgado** [comment]
Hi @MrErlison and @edsoncelio, the translation has been updated to look similar to the original.

File: content/pt-br/platform-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+tags: ["fundamentos", "plataforma", ""]
+---
+
+## O que é
+
+Plataforma como serviço, ou PaaS, é uma plataforma externa para equipes de desenvolvimento de aplicativos implantarem e executarem seus aplicativos. Heroku, Cloud Foundry, App Engine são exemplos de ofertas de PaaS.
+
+## Problema relacionado
+
+Para aproveitar os padrões nativos da nuvem, como [microsserviços](https://glossary.cncf.io/microservices/) ou [aplicativos distribuídos](https://glossary.cncf.io/distributed-apps/), as equipes de operações e os desenvolvedores precisam ser capazes de descarregar uma grande quantidade de operações e trabalho de manutenção. Isso inclui tarefas como provisionamento de infraestrutura, manipulação de [descoberta de serviço](https://glossary.cncf.io/service-discovery/), balanceamento de carga e [escalabilidade](https://glossary.cncf.io/pt-br/scalability/) de aplicativos.
```
Comment (edsoncelio) [inline content/pt-br/platform-as-a-service.md:14]: For the links you can use just:
`[microsserviços](/microservices/)` instead of the full URL.

**Review by edsoncelio — CHANGES_REQUESTED** [review]

File: content/pt-br/platform-as-a-service.md (hunk 1 above)
Comment (juliafmorgado) [reply content/pt-br/platform-as-a-service.md]: Done on all links

**Review by edsoncelio — APPROVED** [review]

**Comment by edsoncelio** [comment]
PTAL @MrErlison @brunoguidone @jessicalins

**Review by jessicalins — COMMENTED** [review]
thanks! I think looks good - I'd just replace aplicativo(s) -> aplicação(ões), to keep the pattern we've been using so far (e.g. https://deploy-preview-1468--cncfglossary.netlify.app/pt-br/stateful-apps/ https://deploy-preview-1468--cncfglossary.netlify.app/pt-br/cloud-native-apps/)
cc @edsoncelio @brunoguidone @MrErlison  WDYT?

File: content/pt-br/platform-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Plataforma como serviço (PaaS)
+status: Completed
+category: Tecnologia
```
Comment (MrErlison) [inline content/pt-br/platform-as-a-service.md:4]: Minor: Tecnologia to tecnologia

**Review by MrErlison — APPROVED** [review]

**Review by jessicalins — APPROVED** [review]
LGTM :)

**Pull request merged**